CheckEngine/Emissions/EmissionReport/Checks/AppendixDEStatusChecks.cs

```diff
@@ -1971,8 +1971,28 @@ public  void CheckExistenceOfValidFuelFlowTest(cCategory Category, out bool Vali
             }
         }
 
-
         /// <summary>
+        /// Determine whether the passed quarter is an ozone season quarter for the current location.
```

Comment:
Small point but not in same relative position as the ECMP 1.0 source.  This will add work to future comparisons if they occur.